salt/_pillar/metalk8s.py

```diff
@@ -163,6 +173,19 @@ def _load_iso_path(config_data):
     return res
 
 
+def _load_kubernetes(config_data):
+    """Load Kubernetes information from BootstrapConfiguration"""
+    kubernetes_data = config_data.get("kubernetes", {})
+
+    # CoreDNS affinity default to soft podAntiAffinity on hostname
+    kubernetes_data["coreDNS"].setdefault("affinity", {}).setdefault(
+        "podAntiAffinity", {}
+    ).setdefault("soft", [{"topologyKey": "kubernetes.io/hostname"}])
+    kubernetes_data["coreDNS"].setdefault("replicas", 2)
```

Comment:
And if the `coreDNS` entry does not exist ? :)

Reply:
Oh right my bad